data.sql

```diff
@@ -162,4 +162,8 @@ FULL JOIN specializations ON specializations.vets_id = vets.id
 WHERE vets.name = 'Maisy Smith' 
 GROUP BY S 
 ORDER BY CNT 
-DESC LIMIT 1;
\ No newline at end of file
+DESC LIMIT 1;
+
+--Performance Improvement--
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215529593-5ea733f7-cb05-4cdf-b58c-32d48e88eceb.png)

- [ ] In this section, you should have acknowledged someone you think should be acknowledged. It would be nice if you Acknowledge **Microverse** :+1: